subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AbstractSmokeTest.groovy

```diff
@@ -112,78 +112,78 @@ abstract class AbstractSmokeTest extends Specification {
         static grgit = "4.1.1"
 
         // https://plugins.gradle.org/plugin/com.github.ben-manes.versions
-        static gradleVersions = "0.46.0"
+        static gradleVersions = "0.47.0"
 
         // https://plugins.gradle.org/plugin/org.gradle.playframework
-        static playframework = "0.13"
+        static playframework = "0.14"
 
         // https://plugins.gradle.org/plugin/net.ltgt.errorprone
         static errorProne = "3.1.0"
 
         // https://plugins.gradle.org/plugin/com.google.protobuf
-        static protobufPlugin = "0.9.2"
+        static protobufPlugin = "0.9.3"
 
         // https://central.sonatype.com/artifact/com.google.protobuf/protobuf-java/4.0.0-rc-2/versions
-        static protobufTools = "3.22.3"
+        static protobufTools = "3.23.4"
 
         // https://plugins.gradle.org/plugin/org.gradle.test-retry
-        static testRetryPlugin = "1.5.2"
+        static testRetryPlugin = "1.5.3"
 
         // https://plugins.gradle.org/plugin/com.jfrog.artifactory
-        static artifactoryPlugin = "4.31.9"
+        static artifactoryPlugin = "4.33.1"
 
         // https://docker.bintray.io/ui/packages/gav:%2F%2Forg.artifactory.oss.docker:artifactory-oss-docker?name=artifactory-oss&type=packages
```

Comment:
Should we remove this completely given that Jcenter is to be sunsetted and the plugin is deprecated?